src/Hl7.Fhir.Core/Rest/Legacy/LegacyFhirClient.cs

```diff
@@ -174,18 +173,21 @@ public ParserSettings ParserSettings
             get => Settings.ParserSettings;
             set => Settings.ParserSettings = value;
         }
-        
+
         /// <summary>
         /// Returns the HttpWebRequest as it was last constructed to execute a call on the FhirClient
         /// </summary>
-        public HttpWebRequest LastRequest { get { return LastClientRequest as HttpWebRequest; } }
+        /// 
+        [Obsolete("LastRequest was already disposed, so no point in having them around", true)]
```

Comment:
Or should we just remove these?  At least they should throw instead of returning null.

Reply:
The true at the end of Obsolete will cause a compiler warning, so that's good enough.